HW1/task-1/index.js

```diff
@@ -0,0 +1,4 @@
+let a = 'Hello JS';
+alert(a);
+let b = 'JS master';
+alert(a = b);
```

Comment:
```suggestion
alert(a == b);
```
`a =  b` означает присвоение